deps/GraphBLAS/Config/GraphBLAS.h.in

```diff
@@ -7848,6 +7848,7 @@ typedef struct
 	int64_t p ;             // Number of none zero values in current column
 	int64_t row_idx ;       // Index of current row
 	GrB_Index nrows ;       // Total number of rows in matrix
+	size_t size ;           // Size of an entry in A
```

Comment:
Why add this if it can be computed from A?

Reply:
we don't want to compute this on every potential call to `next`